### ISSUE TITLE
Errors encountered when compile custom model using caffe xilinx framework

### ISSUE THREAD

**Issue by lebacuong** [Xilinx/Vitis-AI#416]
Hi all,

I am trying to deploy my simple model to ZCU104 BOARD using the caffe framework. After i had quantized my float model, i tried to compile my deploy model to the xmodel using vai_c_caffe command. However, there are some errors encountered according to the local variable referenced before assignment.I tried to fix the error with my knowledge but it didn't work so I want to ask you how to solve this. My float model, deploy model and errors I have attached are in the following [link.](https://drive.google.com/file/d/1SdOwA1OLhPTYWgtOPz1BN3HyogQZYbTu/view?usp=sharing)
Thanks you !

**Comment by lebacuong** [comment]
Hi all,

I am trying to deploy my simple model to ZCU104 BOARD using the caffe framework. After i had quantized my float model, i tried to compile my deploy model to the xmodel using vai_c_caffe command. However, there are some errors encountered according to the local variable referenced before assignment.I tried to fix the error with my knowledge but it didn't work so I want to ask you how to solve this. My float model, deploy model and errors I have attached are in the this [link.](https://drive.google.com/file/d/1CW4iMRXUKqKV5dcnOv7MFFxKwdb9BN5W/view?usp=sharing)
Thanks you !
